docs/topics/modules/about-analytics-integration-cookbook.adoc

```diff
@@ -0,0 +1,18 @@
+[id="about_analytics_integration_cookbook"]
+= About Fabric8-Analytics integration cookbook
+
+Fabric8-Analytics is an open source analytics platform aimed primarily at assisting developers in getting insights and recommendations on the stacks used for developing applications. This platform can be used to get an overview and recommendations for single application components as well.
+
+This cookbook is written for developers who are interested in integrating https://github.com/fabric8-analytics/[Fabric8-Analytics] _stack-analyses_ or _component-analyses_ services within their applications.
+
+The integration cookbook runs you through a couple of case-studies, which describe its usage and describes the other API resources you can call to use Fabric8-Analytics.
```

Comment:
runs you --> guides you, introduces you to, ...

Generally, do not use phrasal verbs in docs.